docs/schema/guidance/repr-beneficial-ownership.rst

```diff
@@ -0,0 +1,75 @@
+.. _representing-bo:
+
+Representing beneficial ownership
+========================================
+
+.. highlights::
+
+    **Key requirement:** If a person is a beneficial owner of an entity - whether directly or indirectly - and the person or entity is required to declare this beneficial ownership, there MUST be an Ownership-or-control Statement connecting the two which represents the beneficial ownership relationship.
+
+
+Overview
+------------------------
+
+Beneficial owners can exercise ownership-or-control *directly* in a company (entity) or *indirectly*, via intermediary entities and arrangements. It must be clear in a BODS dataset which people are declared as beneficial owners of which companies (and which companies are intermediaries). And it must be clear what control or ownership beneficial owners ultimately have (whether it is direct or indirect).
```

Comment:
I don't think we have an agreed way of talking about 'entity', 'company' and 'arrangement'. My suggestion would be that in the general case we should refer to 'entity' and only use terms such as 'company' and 'arrangement' when referring to a specific case.

Here, I think that "intermediary entities and arrangements" has a redundancy because an arrangement is a type of entity.

My preference would be to rewrite the first sentence like this, or similar;

"Beneficial owners can exercise ownership-or-control *directly* in an entity (expected to be a company) or *indirectly*, via intermediary entities (such or arrangements or other companies)."

This would remove the redundancy and there would be a consistency in that we are always using the general term 'entity' first and then giving examples in the brackets that follow.

In the second sentence, I'd prefer us to write 

" ..... beneficial owners of which companies (and which entities are intermediaries)."

In the third sentence, I think that "(whether it is direct or indirect)" can be construed as meaning either:

It must be clear whether control or ownership is direct or indirect
or
It must be clear what the ultimate beneficial ownership is regardless of whether it is direct or indirect

Do we mean either or both of these?

Reply:
I've adopted your suggestions. (And tried to make the third sentence clearer.)

Reply:
You copied my error! It should be "(such as arrangements .....). I'd also prefer "regardless of whether". "regardless whether" is grammatically correct, but it's not in common usage any more.

Reply:
Fixed